crates/dreammaker/src/ast.rs

```diff
@@ -1072,7 +1072,7 @@ pub enum Term {
     /// A use of the `call_ext()()` primitive.
     ExternalCall {
         library_name: Box<Expression>,
-        function_name: Box<Expression>,
+        function_name: Option<Box<Expression>>,
```

Comment:
Maybe these should be renamed. It's not really a "library name" in the single-argument form.

Reply:
what would be a good name for it?
or would it perhaps be a good idea to just make a separate `ExternalCall` and `LoadedExternalCall` variants?